src/gui/DatabaseWidget.cpp

```diff
@@ -326,6 +327,20 @@ void DatabaseWidget::cloneEntry()
     return;
 }
 
+void DatabaseWidget::getTotp()
+{
+    Entry* currentEntry = m_entryView->currentEntry();
+    if (!currentEntry) {
+        Q_ASSERT(false);
+        return;
+    }
+
+    TotpDialog* totpDialog = new TotpDialog(this, currentEntry);
+    totpDialog->setWindowFlags(totpDialog->windowFlags() | Qt::WindowStaysOnTopHint);
+    totpDialog->show();
```

Comment:
Actually I was trying to do this before but couldn't find how since I have no experience with QT. Looks even better now :)